spec/features/admin_views_organization_spec.rb

```diff
@@ -0,0 +1,58 @@
+require "rails_helper"
+
+feature "Admin views organization" do
+  scenario "admin does not see organization details initially", :js do
+    admin = create(:admin)
+    organization = create(:organization)
```

Comment:
Useless assignment to variable - `organization`. Did you mean `organizations_path`?